### PR TITLE
Added player resolver to playerSession in order to fetch further play…

### PR DESCRIPTION
**Pull request by tomarrell**
…er info in one call

Have pre-deployed from this branch so I can continue building out frontend.

### CODE REVIEW

File: graphql_schema.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ schema {
 	}
 
 	type PlayerSession {
-		playerId: ID!
```
Comment (jackyzhen) [inline graphql_schema.go]: I don't think you should only allow getting a playerid via the player linked entity

File: graphql_resolver.go [hunk 2]
```diff
@@ -206,10 +214,12 @@ func (s *SessionResolver) PlayerSessions() ([]*PlayerSessionResolver, error) {
 
 // PlayerSessionResolver struct
 type PlayerSessionResolver struct {
+	player    PlayerResolver
 	playerID  graphql.ID
```
Comment (jackyzhen) [inline graphql_resolver.go]: if you're gonna get rid of playerId from the schema then you don't need this. You'll also need to delete the resolver method for that field. I don't recommend you delete the playerID field though.

File: graphql_resolver.go [hunk 3]
```diff
@@ -190,14 +196,16 @@ func (s *SessionResolver) PlayerSessions() ([]*PlayerSessionResolver, error) {
 
 	sr := make([]*PlayerSessionResolver, len(pSessions))
 
-	for i, s := range pSessions {
+	for i, session := range pSessions {
```
Comment (jackyzhen) [inline graphql_resolver.go]: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Comment (tomarrell) [reply graphql_resolver.go]: Huh interesting, hadn't seen that wiki before. Good to know!

**Comment by tomarrell** [comment]
Have addressed the comments cheers. Will go ahead with the merge

**Pull request merged**